src-tauri/Cargo.toml

```diff
@@ -1,10 +1,10 @@
 [package]
 name = "refstudio"
 version = "0.0.0"
-description = "A Tauri App"
+description = "An open source text editor optimized for writing that relies on references."
 authors = ["you"]
```

Comment:
Should we update `version` and `authors` here?

Reply:
AFAIK We don't need to update the version there, so I kept 0.0.0 in order to have a single place where we configure the version. 

I've updated the description bc it felt like a mandatory task. 

@hammer what should we put in the authors?

PS: Honestly, I don't know where these fields are being used currently. Most likely nowhere.